src/main/java/org/gitlab4j/api/UserApi.java

```diff
@@ -820,6 +820,24 @@ public SshKey addSshKey(Long userId, String title, String key) throws GitLabApiE
         return (sshKey);
     }
 
+    /**
+     * Create new key owned by specified user. Available only for admin users.
+     *
+     * <pre><code>GitLab Endpoint: POST /users/:id/keys</code></pre>
+     *
+     * @param userId    the ID of the user to add the SSH key for
+     * @param title     the new SSH Key's title
+     * @param key       the new SSH key
+     * @param expiresAt the expiration date of the ssh key, optional
+     * @return an SshKey instance with info on the added SSH key
+     * @throws GitLabApiException if any exception occurs
+     */
+    public SshKey addSshKey(Long userId, String title, String key, Date expiresAt) throws GitLabApiException {
```

Comment:
Can you make the method `org.gitlab4j.api.UserApi.addSshKey(Long, String, String)` call this new method with a `null` value for the `expiresAt` parameters